### PR TITLE
Support NO_COLOR. Remove MDC support (NCL-5068)

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

**Comment by rnc** [comment]
@thescouser89 @dwalluck Adding `NO_COLOR` support to GME so that if repour sets that environment variable its automatically picked up.

File: cli/src/main/java/org/jboss/gm/cli/Main.java [hunk 1]
```diff
@@ -197,7 +198,7 @@ private void executeGradle() throws ManipulationException {
                     .map(entry -> "-D" + entry.getKey() + '=' + entry.getValue())
                     .collect(Collectors.toSet());
 
-            if (colour) {
+            if (colour && StringUtils.isEmpty(System.getenv("NO_COLOR"))) {
```
Comment (thescouser89) [inline cli/src/main/java/org/jboss/gm/cli/Main.java:201]: There's a typo in the 'color' name :stuck_out_tongue:

Comment (dwalluck) [reply cli/src/main/java/org/jboss/gm/cli/Main.java:201]: Just for fun, it seems we have too: `NO_COLOR`, `build.setColorOutput`, `-DloggingColours=false`. You're not going to be able to get it 100% consistent, anyway.

Comment (dwalluck) [reply cli/src/main/java/org/jboss/gm/cli/Main.java:201]: I thought that both British and Canadian spellings agreed on the "our" ending and I haven't so far suggested that native speakers change the spelling to American, but I wonder if our tools shouldn't support aliases for every occurrence (but I personally haven't felt like making such a patch).

**Review by thescouser89 — APPROVED** [review]
LGTM!

**Pull request merged**